tools/grift/grpc_plugins_generator.patch

```diff
@@ -0,0 +1,2417 @@
+From 0894590b5020c38106d4ebb2291994668c64f9dd Mon Sep 17 00:00:00 2001
```

Comment:
@nicolasnoble 's suggestion is the right approach : let's do the patch as an interim solution to get something usable and then figure out how to send it upstream after some time and use. I think Nico is trying to make sure that we can use this code longer term as thrift itself gets upgrades (for example, if the thrift issue related to streaming gets solved)